### PR TITLE
fix(broker) : update broker retry interval default value and description (22.04)

### PR DESCRIPTION
**Pull request by AssanDiomande**
## Description

update broker retry interval database

**Fixes** MON-4949

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.10.x
- [ ] 21.04.x
- [x] 21.10.x
- [x] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

* After creating a new poller check in broker configuration if the circular help button mention the 30 sec for retry interval default value

### CODE REVIEW

File: www/install/php/Update-21.10.0.php [hunk 1]
```diff
@@ -18,3 +18,35 @@
  * For more information : contact@centreon.com
  *
  */
+
+
+include_once __DIR__ . "/../../class/centreonLog.class.php";
+$centreonLog = new CentreonLog();
+
+//error specific content
+$versionOfTheUpgrade = 'UPGRADE - 21.10.0: ';
```
Comment (a-launois) [inline www/install/php/Update-21.10.0.php:27]: Shouldn't this be Update-21.10.2 ?

Comment (AssanDiomande) [reply www/install/php/Update-21.10.0.php]: Yes It should be. I've renamed the file.

Comment (AssanDiomande) [reply www/install/php/Update-21.10.0.php]: I've renamed the file

File: www/install/php/Update-21.10.0.php [hunk 2]
```diff
@@ -1,20 +0,0 @@
-<?php
-
```
Comment (a-launois) [inline www/install/php/Update-21.10.0.php:2]: No need to delete this file

**Review by a-launois — DISMISSED** [review]

File: www/install/php/Update-22.04.0-beta.1.php [hunk 3]
```diff
@@ -19,33 +19,45 @@
...
+    $errorMessage = 'Unable to update the description in cb_field';
+    $statement = $pearDB->query("
+        UPDATE cb_field
+        SET `description` = 'Time in seconds to wait between each connection attempt (Default value: 30s).'
+        WHERE `cb_field_id` = 31
+    ");
 } catch (\Exception $e) {
+    if ($pearDB->inTransaction()) {
+        $pearDB->rollBack();
+    }
```
Comment (kduret) [inline www/install/php/Update-22.04.0-beta.1.php:54]: there is no transation

File: lang/fr_FR.UTF-8/LC_MESSAGES/help.po [hunk 4]
```diff
@@ -303,8 +303,8 @@ msgstr "Temps en secondes à attendre avant d'activer le basculement."
 
 #: centreon-web/help.php:50
 #: centreon-web/www/include/configuration/configCentreonBroker/help.php:88
-msgid "Time in seconds to wait between each connection attempt."
-msgstr "Temps d'attente en secondes entre chaque tentative de connexion."
+msgid "Time in seconds to wait between each connection attempt (Default value: 30s)."
+msgstr "Temps d'attente en secondes entre chaque tentative de connexion (Valeur par défaut 30s)."
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/help.po:307]:
```suggestion
msgstr "Temps d'attente en secondes entre chaque tentative de connexion (Valeur par défaut: 30s)."
```

File: www/install/php/Update-22.04.0-beta.1.php [hunk 5]
```diff
@@ -46,6 +56,5 @@
         " - Error : " . $e->getMessage() .
         " - Trace : " . $e->getTraceAsString()
     );
-
```
Comment (kduret) [inline www/install/php/Update-22.04.0-beta.1.php:49]: retrieve blank line please
do not compress the code

**Review by kduret — APPROVED** [review]

**Review by a-launois — APPROVED** [review]

**Pull request merged**